src/servers/src/http/event.rs

```diff
@@ -514,7 +514,7 @@ pub async fn loki_ingest(
             let line = entry.line;
 
             // create and init row
-            let mut row = Vec::with_capacity(schemas.capacity());
+            let mut row = Vec::with_capacity(schemas.len());
             for _ in 0..row.capacity() {
```

Comment:
the key is to change this `row.capacity()` to schemas_len

Reply:
I will change `row.capacity()` to `schemas.len()`